lib/fs/storage.go

```diff
@@ -91,6 +92,35 @@ func (s Storage) WalkWithContext(_ context.Context, path string, callback func(p
 	return s.Walk(path, callback)
 }
 
+// Copy copies a file.
+func (s Storage) Copy(src string, dst string, options ...map[string]interface{}) error {
+	mode := 0644
+
+	for _, opt := range options {
+		if v, ok := opt["mode"]; ok {
+			if m, ok := v.(int); ok {
+				mode = m
+			}
+		}
+	}
+
+	input, err := ioutil.ReadFile(src)
+	if err != nil {
+		return err
+	}
+
+	if err := ioutil.WriteFile(dst, input, fs.FileMode(mode)); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
You can simplify this by returning the result of `ioutil.WriteFile`.

```suggestion
	return ioutil.WriteFile(dst, input, fs.FileMode(mode))
```